magento/magento2
# Aria-atomic="true" missing on error container

### Description (*)
Issue : Aria-atomic="true" is missing on error container

Summary: The page contains an error container made with the WAI-ARIA `role="alert"` attribute, but it does not contain a WAI-ARIA attribute that will make sure all assistive technology can read the error message after more than one invalid submission.

Fix: To make sure that all assistive technologies will read the error message after more than one invalid submission, added the WAI-ARIA attribute `aria-atomic="true"` to the error container tag.

### Related Pull Requests
N/A
### Fixed Issues (if relevant)
N/A

**Precondition:**

have Cart price rule with discount code;

**Manual testing scenario:**

1. Go to Storefront;
2. Add product to cart;
3. Go to Shopping Cart;
4. Expand **Apply Discount Code** tab and fill correct code

**Expected Result:** :heavy_check_mark:  The success message is shown on the page

![Screenshot from 2020-09-03 11-19-16](https://user-images.githubusercontent.com/51679138/92090277-e8efc280-edd7-11ea-8724-e7f8b1a456cc.png)

5. Try to fill with incorrect code

**Expected Result:** :heavy_check_mark:  The error message is shown on the page

![Screenshot from 2020-09-03 11-19-06](https://user-images.githubusercontent.com/51679138/92090317-f5741b00-edd7-11ea-98ea-29f60f2ae0be.png)

6. Proceed to Checkout;
7. Fill all required fields and go to  Review & Payments step;
8. Expand the **Apply Discount Code** tab and fill correct code

**Expected Result:** :heavy_check_mark:  The success message is shown on the page

![Screenshot from 2020-09-03 11-13-30](https://user-images.githubusercontent.com/51679138/92089329-b1cce180-edd6-11ea-9f2b-d557596ceb6d.png)

9. Try to fill with incorrect code

**Expected Result:** :heavy_check_mark:  The error message is shown on the page

![Screenshot from 2020-09-03 11-13-19](https://user-images.githubusercontent.com/51679138/92089250-8fd35f00-edd6-11ea-8c42-82291df262ae.png)
### Questions or comments
N/A

### Contribution checklist (*)



### Resolved issues:
1. [x] resolves magento/magento2#29560: Aria-atomic="true" missing on error container
